backend/middleware/notificationHandler.js

```diff
@@ -0,0 +1,47 @@
+const moment = require('moment')
+const { INLINES, MARKS } = require('@contentful/rich-text-types')
+const { documentToHtmlString } = require('@contentful/rich-text-html-renderer')
+
+const options = {
+  renderMark: {
+    [MARKS.BOLD]: text => `<p class="govuk-!-font-weight-bold">${text}</p>`,
```

Comment:
<strong>${text}</strong> would be better here I reckon.  We do the same in content.js